contracts/Shortfall/Shortfall.sol

```diff
@@ -279,6 +283,19 @@ contract Shortfall is Ownable2StepUpgradeable, AccessControlled, ReentrancyGuard
         _startAuction(comptroller);
     }
 
+    /**
+     * @notice Updates the inventive BPS
+     * @param _incentiveBps New incentive BPS
+     * @custom:event Emits IncentiveBpsUpdated on success
+     * @custom:access Restricted to owner
+     */
+    function updateIncentiveBps(uint256 _incentiveBps) external onlyOwner {
```

Comment:
I would integrate this with the ACM instead of using `onlyOwner`

Reply:
I've integrated this with ACM. I also added commits to put the other setter functions behind the ACM to avoid another PR and conflicts